meetings/2023-09-19.md

```diff
@@ -0,0 +1,140 @@
+* TBL: Would shapetrees be possible to describe achat?
+* eP: For example, in the chat spec you have an directory with chats according to a shape. In shapetrees we can actually link to that shape in a follow-your-nose approach.
+* TBL: So when you're looking for something in a pod, does it then declare what shape it is?
+* eP: In SAI we use data registries which link data to a shapetree which contain shapes.
+* TBL: Do shapetrees have uri templates?
+* WT: Yes, you can specify the name of a path.
+
+---
+
+* eP: What did you say about the app has an id? ... this seems to clash with colid-OIDC?
+* TBL: For example if I have a timbl-bot, it has a has a pod and can run tasks like importing matrix chats etc. If you look at [timblebot](https://timblbot.inrupt.net/profile/card#me) is a bot I control. I use WAC to give its access as an agent to my pod and it just logs in. It's not an OIDC flow.
+* eP: We have a separate [issue](https://github.com/solid/specification/issues/504) about server-side clients. We had a talk about that two weeks ago, and we agree there needs to be a better way. We just assume clients have their own ID.
+* TBL: I agree that from the point of view of CSS it is indistinguishable as a person.
+* eP: We also want to offer cases where different users could use the same bot.
+* TBL: You could imagine it creating a new agent/WebID specifically for the task of a specific user.
+
+---
+
+* eP: I want to ask michal/mrkvon and Reza to share their opinion.
+* RS: I was just listening for a while. You mentioned the issue about a bot that users can use to access a pod. ... I do not know specifically what I can say here.
+* eP: I would be interested in requirements and use cases.
+* RS: Let me talk to Michael and come back to you.
+* RS: Is there any way we can authorize the application on itself?
+* eP: Yes, [CSS has a way](https://communitysolidserver.github.io/CommunitySolidServer/6.x/usage/client-credentials/), it is not standard though. There is also the issue (linked above) in which we are talking about alternative flows, and there is a similar [document](https://github.com/solid/solid-oidc/blob/main/alternative-flows.md) in the solid-oidc repo.
+* mrkvon: I came to expect a bit of talks about the solid chat. I have one specific issue.  I think the current approach is that all messages are stored in one persons pod. I find the [approach](pod-chat.com) Michiel hinted at of having messages in multiple peoples messages more equal. We use that for an implementation (https://sleepy.bike/).
+* eP: I think there are advantages, but it might impact replication, e.g. when there is only one copy of a message and a pod is down we would lose part of the conversation. 
```

Comment:
```suggestion
* eP: I think there are advantages, but it might impact replication, e.g., when there is only one copy of a message, and a pod is down, we would lose part of the conversation. 
```